assets/css/default.css

```diff
@@ -432,17 +432,30 @@ p.video-data   { margin: 0; font-weight: bold; font-size: 80%; }
  * Footer
  */
 
-footer {
-  color: #919191;
+.light-theme footer {
+  color: #7c7c7c;
   margin-top: auto;
   padding: 1.5em 0;
   text-align: center;
   max-height: 30vh;
```

Comment:
There's no need for you to duplicate these styles so many times across the file. You can just place them under a footer element selector once.